Documentation/general/cli-ux-guidelines.md

```diff
@@ -7,6 +7,11 @@ implementation details. The main goal of the document is to provide guidelines
 for new authors that wish to make pull requests and add new functionalities and
 commands to the .NET Core command line tools (CLI).
 
+The guidelines presented in this document have been adopted to provide a clear and concise 
+command line syntax that is easy to learn, remember and work with, and that has an added benefit 

```

Comment:
> has an added benefit

This part sounds like you are trying to sell a juicer. If I wait closer to the end of the show, do I get two for one?